### PR TITLE
Config: Improve user input field names

### PR DESCRIPTION
**Pull request by orelmisan**
Improve the naming of the user-facing input fields:
`trafficGeneratorImage`                      -> `trafficGenContainerDiskImage`
`trafficGeneratorNodeSelector`          -> `trafficGenTargetNodeName`
`trafficGeneratorPacketsPerSecond` -> `trafficGenPacketsPerSecond`
`vmContainerDiskImage`                     -> `vmUnderTestContainerDiskImage`
`DPDKLabelSelector`                             -> `vmUnderTestTargetNodeName`
`portBandwidthGB`                               -> `portBandwidthGbps`

The internal names are changed to match.

The output fields will be changed in a follow-up PR.

### CODE REVIEW

File: pkg/internal/config/config.go [hunk 1]
```diff
@@ -31,24 +31,24 @@ import (
...
 
 const (
-	TrafficGenDefaultContainerDiskImage     = "quay.io/kiagnose/kubevirt-dpdk-checkup-traffic-gen:main"
-	TrafficGeneratorPacketsPerSecondDefault = "14m"
-	VMContainerDiskImageDefault             = "quay.io/kiagnose/kubevirt-dpdk-checkup-vm:main"
-	TestDurationDefault                     = 5 * time.Minute
-	PortBandwidthGBDefault                  = 10
-	VerboseDefault                          = false
+	TrafficGenDefaultContainerDiskImage = "quay.io/kiagnose/kubevirt-dpdk-checkup-traffic-gen:main"
+	TrafficGenDefaultPacketsPerSecond   = "8m"
```
Comment (RamLavi) [inline pkg/internal/config/config.go]: I think the value change deserved a separate PR

File: README.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ roleRef:
 | spec.param.vmUnderTestContainerDiskImage   | VM under test container disk image                                     | False        | Defaults to `quay.io/kiagnose/kubevirt-dpdk-checkup-vm:main`          |
 | spec.param.vmUnderTestTargetNodeName       | Node Name on which the VM under test will be scheduled to              | False        | Assumed to be configured to Nodes that allow DPDK traffic             |
```
Comment (RamLavi) [inline README.md:76]: > Bandwidth units are multiplier * bits/unit of time.

I don't think is accurate. 
This parameter talks about the capacity of the wire, not the speed of packets inside it.
It's like talking about a physical pipe. the dimensions of the pipe are equivalent to the bandwidth (For example, a pipe can be 4 Zoll/Inches in diameter), and Bps is talking about how much fluid (or for our example, let's say "packets" of fluid) can run through it - could be a lot of small packets, or a little big packets. But the dimension is the same.

This parameter is passed directly to the trex_cfg.yaml file (see [manual](https://trex-tgn.cisco.com/trex/doc/trex_manual.html#trex_config_yaml_config_file)), to the `port_bandwidth_gb` param, and is used by trex to know how much memory it needs to allocate.

I would suggest `PortBandwidthGb`

Comment (RamLavi) [reply README.md:76]: I stand corrected, Bandwidth units are in bps

File: pkg/internal/config/config.go (hunk 1 above)
Comment (orelmisan) [reply pkg/internal/config/config.go]: Reverted this change.

**Comment by orelmisan** [comment]
[Change](https://github.com/kiagnose/kubevirt-dpdk-checkup/compare/d33a4c4865aada9cdd534aeedf6f211bf30b1ee1..fb99c8f1a9b6b82b766b580e9ceafb3c953d9c74): Answered review [comment](https://github.com/kiagnose/kubevirt-dpdk-checkup/pull/168#discussion_r1275844146).

**Comment by orelmisan** [comment]
Tested against an OpenShift Virtualization 4.14 cluster.

**Review by RamLavi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**